packages/flutter/lib/src/material/switch.dart

```diff
@@ -156,7 +156,7 @@ class _SwitchState extends State<Switch> with TickerProviderStateMixin {
     Color inactiveTrackColor;
     if (widget.onChanged != null) {
       inactiveThumbColor = widget.inactiveThumbColor ?? (isDark ? Colors.grey.shade400 : Colors.grey.shade50);
-      inactiveTrackColor = widget.inactiveTrackColor ?? (isDark ? Colors.white30 : Colors.black26);
+      inactiveTrackColor = widget.inactiveTrackColor ?? (isDark ? Colors.white30 : const Color(0x20000000));
```

Comment:
It would help to include a comment that explains that const Color(0x20000000) is the same as Colors.black.withOpacity(0.32). In fact you define a black32 constant here and use that. That's assuming that black32 isn't going to be needed elsewhere; in that case we might as well defined Colors.black32

Reply:
Done!